Multisig/UI/Settings/Safe Settings/AdvancedSafeSettingsView.swift

```diff
@@ -40,15 +40,12 @@ struct AdvancedSafeSettingsView: View {
     }
     
     var fallbackHandlerView : some View {
-        let fallbackHandler = safe.fallbackHandler?.checksummed ?? ""
         let title = App.shared.gnosisSafe.fallbackHandlerLabel(fallbackHandler: safe.fallbackHandler)
-        return Group {
-            if fallbackHandler.isEmpty || fallbackHandler == "0" {
+        return Section(header: SectionHeader("FALLBACK HANDLER")) {
+            if safe.fallbackHandler?.isEmpty ?? true {
```

Comment:
the function to check that could be something like

```swift
.. GnosisSafe {

  ...

  func hasFallbackHandler(at address: Address?) -> Bool {
     address?.isZero == false 
  }

}
```

and then in this view file:

```swift
   if App.shared.gnosisSafe.hasFallbackHandler(at: safe.fallbackHandler) {
        AddressCell(...)
   else {
      Text("Not set")
  }
```